.gitmodules

```diff
@@ -5,3 +5,6 @@
 [submodule "clamavj"]
 	url = https://github.com/UNC-Libraries/clamavj.git
 	path = clamavj
+[submodule "admin-permissions-editor"]
+	path = admin-permissions-editor
```

Comment:
this is totally beyond picky of me, but would you mind flipping the order of path and url to match the other entries? :-)